src/Mod/PartDesign/CMakeLists.txt

```diff
@@ -55,6 +55,11 @@ set(PartDesign_TestScripts
     PartDesignTests/TestInvoluteGear.py
 )
 
+set(PartDesign_TestFixtures
+    PartDesignTests/Fixtures/InvoluteGear_v0-20-1.FCStd
```

Comment:
Can you please use a generic name. These files will stay also for FC 0.21 or 1.0

Reply:
The name is on purpose. Those files are used by unit tests to ensure we can 
a) load the old files
b) recalculate them successfully
b) come to the same result as with earlier versions. 

So if I have a unit test, asserting compat with v0.20.1, I do not want a generic name, but a name that makes clear that those are files from v0.20.1

Reply:
Well, only experts will understand this. Especially the reference to 0.20.1 (the ".1") is a problem. Here you add a new feature, so it will not be backported. Therefore, if you insist to include the version name, please use then only 0.20.

Reply:
I feel we still have a misunderstanding here. This is not about backporting, it's about ensuring backward compatibility.

I wrote automated tests that ensure the current version (0.21-alpha, 1.0, what ever the future brings) is still able to correctly handle gears created with FreeCAD 0.20.1. That was my safety net during development and kinda insurance for my older projects against potential regressions in the future.

As the gear's code hasn't changed in the recent past, it shouldn't really matter whether I'd use 0.20.0, 0.20.1 or 0.20.2 -- I just happen to have a 0.20.1 around to create those reference files. And because it was a 0.20.1 I suffixed them with "_v0-20-1".

Is your proposal to recreate the reference files with the original 0.20.0 and name them "..._v0-20" instead? Or is having the patch level in file name the problem? If so, why?
Or is it still unclear what I want to achieve here?

Reply:
(Answering to [this comment](https://github.com/FreeCAD/FreeCAD/pull/8184#issuecomment-1399381491), which somehow got out of context)

> > As the gear's code hasn't changed in the recent past, it shouldn't really matter whether I'd use 0.20.0, 0.20.1 or 0.20.2 -- I just happen to have a 0.20.1 around to create those reference files. And because it was a 0.20.1 I suffixed them with "_v0-20-1".
> 
> But this is my point. the "-1" implies for other or future developers that there was a change in the 0.20 code line. But since this was not the case, the naming can be confusing (a future developer might spend time to onvestigate what might happened between 0.20.0, 0.20.1 0.20.2 etc.)
> 
> Therefore in my opinion the naming should be just "_v0-20".

This is a reasoning I can follow. So I kept the version number in the file name, but removed patch-level.